app/src/main/java/com/epfl/beatlink/ui/search/SearchBarScreen.kt

```diff
@@ -0,0 +1,128 @@
+import com.epfl.beatlink.ui.theme.PrimaryPurple
+import com.epfl.beatlink.ui.theme.PrimaryRed
+
+@Composable
+fun SearchBarScreen(navigationActions: NavigationActions) {
+  val selectedCategory = remember { mutableStateOf("Songs") }
+  val recentSearches = remember { mutableStateOf(listOf("Song 1")) }
+
+  Scaffold(
+      topBar = { ShortSearchBarLayout(navigationActions) },
+      bottomBar = {
+        BottomNavigationMenu(
+            onTabSelect = { route -> navigationActions.navigateTo(route) },
+            tabList = LIST_TOP_LEVEL_DESTINATION,
+            selectedItem = navigationActions.currentRoute())
+      },
+      modifier = Modifier.testTag("searchBarScreen")) { paddingValues ->
+        Column(
+            modifier =
+                Modifier.testTag("recentSearchesColumn")
+                    .fillMaxSize()
+                    .padding(paddingValues)
+                    .background(color = Color.White)) {
+              HorizontalDivider(
+                  color = Color.LightGray, thickness = 1.dp, modifier = Modifier.testTag("divider"))
+
+              Spacer(modifier = Modifier.testTag("spacer").height(8.dp))
+
+              Row(
+                  horizontalArrangement = Arrangement.SpaceEvenly,
+                  verticalAlignment = Alignment.CenterVertically,
+                  modifier =
+                      Modifier.testTag("categoryRow")
+                          .fillMaxWidth()
+                          .padding(horizontal = 12.dp, vertical = 8.dp)) {
+                    CategoryButton("Songs", selectedCategory.value, PrimaryRed) {
+                      selectedCategory.value = "Songs"
+                      recentSearches.value = listOf("Song A", "Song B", "Song C")
+                    }
+                    CategoryButton(
+                        "Events", selectedCategory.value, Color(0xFFFFA500)) { // Orange color
+                          selectedCategory.value = "Events"
+                          recentSearches.value = listOf("Event A", "Event B", "Event C")
+                        }
+                    CategoryButton(
+                        "People", selectedCategory.value, PrimaryPurple) { // Purple color
+                          selectedCategory.value = "People"
+                          recentSearches.value = listOf("Person A", "Person B", "Person C")
+                        }
+                  }
+
+              Spacer(modifier = Modifier.testTag("spacer").height(22.dp))
+
+              StandardLazyColumn(title = "RECENT SEARCHES", list = recentSearches.value)
+            }
+      }
+}
+
+@Composable
+fun CategoryButton(
+    category: String,
+    selectedCategory: String,
+    categoryColor: Color,
+    onClick: () -> Unit
+) {
+  val isSelected = selectedCategory == category
+  val backgroundColor = if (isSelected) categoryColor else Color.Transparent
+  val contentColor = if (isSelected) Color.White else categoryColor
```

Comment:
here you should use the app backgroup (eg. lightThemeBackground) instead of plain white (it won't change much but it is for consistency)